requirements/ingest/gcs.txt

```diff
@@ -35,37 +35,19 @@ frozenlist==1.4.1
     # via
     #   aiohttp
     #   aiosignal
-fsspec==2024.3.1
+fsspec==2024.5.0
     # via
+    #   -c ./ingest/../deps/constraints.txt
     #   -r ./ingest/gcs.in
     #   gcsfs
-gcsfs==2024.3.1
+gcsfs==0.8.0
```

Comment:
Not sure where this `0.8.0` is coming from, don't see that in the [version release history](https://pypi.org/project/gcsfs/#history) for that package PyP.